arch/arm/soc/atmel_sam/sam4s/Kconfig.soc

```diff
@@ -4,6 +4,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
+# The default settings in this file were for the board SAM4S Xplained.
```

Comment:
I had put that line in when I was thinking about the FWS value and its VDDCORE and VDDIO. I am removing the line.